core/templates/pages/exploration-editor-page/services/state-tutorial-first-time-backend-api.service.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview StateTutorialFirstTimeService backend api service.
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+import { ServicesConstants } from 'services/services.constants';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class StateTutorialFirstTimeBackendApiService {
```

Comment:
Should we rename this class/file to StateTutorialEventRegistryBackendApiService. (As this API service doesn't know or validates whether the record is happening for the first time.)

Reply:
Done!